From ec4ee2b2c3102c2b32cc45cde88d5aced2ac19e1 Mon Sep 17 00:00:00 2001
From: Yu Jiahua <yujiahua1@huawei.com>
Date: Thu, 20 Jan 2022 12:22:32 +0800
Subject: [PATCH] Revert "sched: Add menuconfig option for
 CONFIG_SCHED_OPTIMIZE_LOAD_TRACKING"
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: ec4ee2b2c3102c2b32cc45cde88d5aced2ac19e1


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4QU5Z?from=project-issue
CVE: NA

--------------------------------

This patch revert ias feature from open-euler kernel.

This reverts commit 6d7dd529725c2b71c4499858cd8fbcfbd69c3b19.

Signed-off-by: Yu Jiahua <Yujiahua1@huawei.com>
Reviewed-by: Chen Hui <judy.chenhui@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 init/Kconfig | 15 +++++++--------
 1 file changed, 7 insertions(+), 8 deletions(-)

diff --git a/init/Kconfig b/init/Kconfig
index dbbf73085919..d2a0ac2e12db 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -774,14 +774,6 @@ config GENERIC_SCHED_CLOCK
 
 menu "Scheduler features"
 
-config SCHED_OPTIMIZE_LOAD_TRACKING
-	bool "Optimize scheduler load tracking"
-	default n
-	help
-	  Optimize scheduler load tracking, when load balance is not important
-	  in system, we close some load tracking in tick and enqueue or dequeue
-	  task, in this way, we can save some unnecessary cpu overhead.
-
 config IAS_SMART_HALT_POLL
 	bool "Enable smart halt poll"
 	default n
@@ -2096,6 +2088,13 @@ endmenu		# General setup
 
 source "arch/Kconfig"
 
+config SCHED_OPTIMIZE_LOAD_TRACKING
+	bool "Optimize scheduler load tracking"
+	default n
+	help
+	  Optimize scheduler load tracking, when load balance is not important
+	  in system, we close some load tracking in tick and enqueue or dequeue
+	  task, in this way, we can save some unnecessary cpu overhead.
 
 config RT_MUTEXES
 	bool
-- 
2.26.2

